### PR TITLE
Desktop version 3

### PR DESCRIPTION
**Pull request by AckonSamuel**
Linters were used to detect programmable errors.
There was a merge conflict between the README.md files of the main and Desktop_version_3 branches. The merger conflict was successfully detected and resolved.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -43,33 +48,61 @@
...
+                       <ul class="hellso">
+                           <div class="tech2"><li class="maybe"><p>HTML/CSS</p></li></div>
+                               <div class="tech2"><li class="maybe"><p>Ruby on rails</p></li></div>
+                                  <div class="tech2"><li class="maybe"><p>+1</p></li></div>
+                       </ul>
+
+           
+                    <div claas="being"> 
+                        <div alt="#" class="project-view viewdesk">
+                            <div class="word worddesk"><p>See this project</p></div>
```
Comment (micheaol) [inline index.html:90]: Please ensure that the `See this project ->` should be a button that takes you to the Live Demo of the featured project or to the GitHub repository.

File: index.html [hunk 2]
```diff
@@ -43,33 +48,61 @@
...
+                <div class="namep namedesk"><h3>Project name goes here</h3></div>
+                <div class="technology techdesk">
+                <ul class="hello hellodesk">
+                    <div class="tech1 tech1desk"><li class="maybe"><p>HTML/CSS</p></li></div>
+                    <div class="tech1 tech1desk"><li class="maybe"><p>Ruby on rails</p></li></div>
+                   <div class="tech1 tech1desk"><li class="maybe"><p>JavaScript</p></li></div>
                 </ul>
                 </div>
-                <div alt="#" class="project-view">
-                    <div class="word"><p>See this project</p></div>
```
Comment (micheaol) [inline index.html:62]: Please ensure that the `See this project ->` should be a button that takes you to the Live Demo of the featured project or to the GitHub repository.

File: index.html [hunk 3]
```diff
@@ -43,33 +48,61 @@
...
+<section class="hallelujah">
     
-    <section class="Works">
-        <div class="Project-name"><h2>Projects</h2></div>
+    <section class="works">
+        <div class="project-name"><h2>Projects</h2></div>
         <div class="work one">
-            <div class="project photo"><img src="assets/work/work_one.png" class="work_one" alt="#"></div>
-            <div class="project summary">
-                <div class="nameP"><h3>Project name goes here</h3></div>
```
Comment (micheaol) [inline index.html:53]: Please ensure you replicate the design as much as possible (e.g., font, colors, images, buttons, margins) [using the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9). Please ensure to align` projects` as in the Figma template.

Your Implementation | Figma Design
:---:|:---:
your-design | figma.

File: index.html [hunk 4]
```diff
@@ -35,6 +39,7 @@
          don’t hesitate and contact me.</p></div>
        <div class="rectangle">
           <div>Start collaboration</div></div>
```
Comment (micheaol) [inline index.html:41]: Please ensure that the `Start collaboration` be a button according to the requirement.

File: index.html [hunk 5]
```diff
@@ -194,17 +214,108 @@ <h4>Frameworks</h4>
...
+    </div>
+    <div class="fooc textmessage">
+        <fieldset>
+            <div class="field">
+                <textarea name="comment" maxlength="500" placeholder="Enter your message here..." required></textarea>
+            </div>
+        </fieldset>
+    </div>
+        <button value="submit" class="collabo">
+            <div>Start collaboration</div></button>
```
Comment (micheaol) [inline index.html:279]: Please note that it is semantically incorrect to put a `<div>` inside of a `<button>`. I also observed that you have some `a` tags with `<div>` inside of them. Please note that this is not correct semantically.

File: index.html [hunk 6]
```diff
@@ -43,33 +48,61 @@
...
-                    <div class="tech1"><li class="maybe"><p>Ruby on rails</p></li></div>
-                   <div class="tech1"><li class="maybe"><p>JavaScript</p></li></div>
+            <div class="project photo"><img src="assets/work/work_one.png" class="work-one" alt="#"></div>
+            <div class="project summary desksum">
+                <div class="namep namedesk"><h3>Project name goes here</h3></div>
+                <div class="technology techdesk">
+                <ul class="hello hellodesk">
+                    <div class="tech1 tech1desk"><li class="maybe"><p>HTML/CSS</p></li></div>
+                    <div class="tech1 tech1desk"><li class="maybe"><p>Ruby on rails</p></li></div>
+                   <div class="tech1 tech1desk"><li class="maybe"><p>JavaScript</p></li></div>
```
Comment (micheaol) [inline index.html:63]: Please note that it is not semantically correct to have `p` tags wrapped inside `li`. Kindly make necessary changes to all where it applies.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @AckonSamuel , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 
- Your repo is professional 
- Well structured files 
- You set up linter files and no errors. 


## Required Changes ♻️

- _Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brytebee — APPROVED** [review]
Hi, @AckonSamuel👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**